### PR TITLE
Season view shouldn't use another view. Removed that view from the qu…

### PR DESCRIPTION
**Pull request by razzeee**
…ery.

As reported in http://trac.kodi.tv/ticket/16387

### CODE REVIEW

File: xbmc/video/VideoDatabase.cpp [hunk 1]
```diff
@@ -401,23 +401,28 @@ void CVideoDatabase::CreateViews()
...
                                      "  min(episode.c%02d) AS aired "
                                      "FROM seasons"
-                                     "  JOIN tvshow_view ON"
-                                     "    tvshow_view.idShow = seasons.idShow"
+                                     "  JOIN tvshow ON"
+                                     "    tvshow.idShow = seasons.idShow"
                                      "  JOIN episode ON"
-                                     "    episode.idShow = seasons.idShow AND episode.c%02d = seasons.season"
+                                     "    episode.idShow = seasons.idShow AND"
+                                     "    episode.c%02d = seasons.season"
```
Comment (phate89) [inline xbmc/video/VideoDatabase.cpp]: `episode.idSeason = seasons.idSeason`
I forgot to update that query to the latest changes

Comment (phate89) [inline xbmc/video/VideoDatabase.cpp]: `episode.idSeason = seasons.idSeason`
I forgot to update that query to the latest changes

Comment (Razzeee) [inline xbmc/video/VideoDatabase.cpp]: Good catch

**Comment by phate89** [comment]
I actually checked again the query and it doesn't work. I remembered why when I fixed the first time this query I didn't use a similar version. A tvshow could have more than one path..So if you use only one query the playcount of the episodes of that tv shows is twice if the tv show has 2 folders, 3 times if it has 3 folders etc etc. We could divide the count result by the count path ids but we will loose the advantage and will make the sqlite query slower (and sqlite doesn't suffer too much a view of a view)

**Comment by phate89** [comment]
I found a solution. We could use:
`COUNT(DISTINCT(CASE WHEN files.playCount > 0 THEN episode.idEpisode END )) AS playCount`
It's still a lot faster in MySQL and slightly faster in SQLite.. We need to check if it's the same with a low power cpu like the raspberry one though.

**Pull request closed without merging**